content/news/playtest-20180101.md

```diff
@@ -0,0 +1,63 @@
+<img src="/images/news/20180101-serverplayers.png" width="600" alt="Server player list">
+
+The new server player list is one of several improvements to the multiplayer UI
+</div>
+
+Of course, we realize that the vast majority of OpenRA players do not play online. We haven't forgotten about you either!
+Improvements for the singleplayer experience include:
+
+ * The full Harkonnen campaign in D2K
+ * Refinements and improvements to several campaign missions in Red Alert and Tiberian Dawn
+ * A fix for the AI cheating when it targets superweapons and support powers
+ * Several other changes that should improve the AI behaviour and performance
+
+<div style="text-align:center" markdown="1">
+![AI Airstrike](/images/news/20180101-airstrike.png)
+
+Improvements to the AI support power targeting mean that this will be a rarer sight.
+ </div>
+
+Other notable changes include:
+
+* New lobby options to disable bounty, build radius, and re-packing Construction Yards
+* Units will no longer target units revealed by the Red Alert GPS Satellite
+* A collection of community-driven balance changes for Red Alert
+* Fixes for many graphical glitches and polish issues
+* Fixes for several platform integration issues on Windows and macOS
+* Further improvements to memory usage and performance
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20180101-mapoptions.png" width="600" alt="Expanded game options">
+
+The Map Options tab includes several new settings, and can be customized by scripted maps.
+</div>
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20180101-winmacos.png" width="600" alt="Windows and macOS improvements">
+
+Platform integration fixes include "Pin to taskbar" support on Windows and 64 bit support for macOS.
+</div>
+
+As always, we suggest reading the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog) for more information and to view the full list of changes.
+We make these playtests available to get feedback from the community about the future direction of OpenRA, and we value your feedback.
+Let us know what you think and please report any bugs you discover via [our forum](http://www.sleipnirstuff.com/forum/viewforum.php?f=80), our [GitHub issue tracker](https://github.com/OpenRA/OpenRA/issues), or in the comments below.
+
+OpenRA playtest builds do not overwrite the release version if you are on Windows or macOS, and you can switch between the two when joining an online server. So head over to our [download page](/download/) and give it a try!
+
+This development cycle brought significant improvements for community mods, but we recommend that modders wait for a future playtest before updating your projects.  We plan to merge a new version of our semi-automated rule upgrader before the final release, which will siimplify the process of updating your [Mod SDK](https://github.com/OpenRA/OpenRAModSDK) based projects to the latest engine version. Stay tuned for more info!
```

Comment:
"that modders wait [...] before updating your projects." 3rd person versus 2nd person? your -> their?